sensor/common/networkflow/manager/manager_impl.go

```diff
@@ -637,7 +662,7 @@ func computeUpdatedProcesses(current map[processListeningIndicator]timestamp.Mic
 	if !env.ProcessesListeningOnPort.BooleanSetting() {
 		if len(current) > 0 {
 			logging.GetRateLimitedLogger().Warn(loggingRateLimiter,
```

Comment:
```suggestion
			logging.GetRateLimitedLogger().WarnL(loggingRateLimiter,
```

Reply:
Oh, that is a crucial mistake! I thought that `WarnL` is like `WarnLn`, but instead it accepts the rate limiter key instead. I think that `WarnL(limiter string, template string, args ...interface{})` should accept some type that is not string for the `limiter` so that such mistakes can be avoided.